test/new_test/new_test.f90

```diff
@@ -174,17 +178,9 @@ function get_command_path() result(prog)
     allocate(character(len=length) :: prog)
     call get_command_argument(0, prog)
     path = dirname(prog)
-    if (get_os_type() == OS_WINDOWS) then
-      prog = join_path(path, "..", "app", "fpm.exe")
-      if (.not.exists(prog)) then
-        prog = join_path(path, "..", "..", "app", "fpm.exe")
-      end if
-    else
-      prog = join_path(path, "..", "app", "fpm")
-      if (.not.exists(prog)) then
-        prog = join_path(path, "..", "..", "app", "fpm")
-      end if
+    prog = join_path(path, "..", "app", "")
+    if (.not.exists(prog)) then
+       prog = join_path(path, "..", "..", "app", "")
     end if
```

Comment:
```suggestion
    prog = join_path(path, "..", "app", "fpm")
    if (.not.exists(prog)) then
       prog = join_path(path, "..", "..", "app", "fpm")
    end if
    if (get_os_type() == OS_WINDOWS) then
      prog = prog // '.exe'
    end if
```

Reply:
Thanks for the review @LKedward. Yes, this approach looks cleaner as it avoids code duplication. I have made the changes accordingly.